### DISCUSSION TITLE
Simulator not working as required

### DISCUSSION THREAD

**Discussion by Kr1ssr1dhar** [dominik-tylczynski/mfs-plc-sim#15] [General]
Hi Dominik,

Attached the issue explained in detail. Kindly help. Let me know if I need to share more information. I will be happy to host a call if you need. Please share me your mail id to send meeting invite.

Thanks & Regards

K. Sridhar
[PLC Emulator Issue.docx](https://github.com/dominik-tylczynski/mfs-plc-sim/files/11598743/PLC.Emulator.Issue.docx)

**Comment by dominik-tylczynski** [comment]
Hello, can you provide your MFS.properties files? I would like to check how you have defined telegram structures there.

**Reply by Kr1ssr1dhar** [reply]
Attached the word with the requested information
[PLC Emulator Issue.docx](https://github.com/dominik-tylczynski/mfs-plc-sim/files/11598961/PLC.Emulator.Issue.docx)

**Comment by dominik-tylczynski** [comment]
Can you also provide a screenshot of the definition of ZMFS_S_TELE_STRUCTURE from SE11?

**Reply by Kr1ssr1dhar** [reply]
Screenshot updated in word doc
[PLC Emulator Issue.docx](https://github.com/dominik-tylczynski/mfs-plc-sim/files/11599062/PLC.Emulator.Issue.docx)

**Reply by Kr1ssr1dhar** [reply]
Any update Dominik from your end on the issue?
